templates/install/project.md

```diff
@@ -55,64 +33,74 @@ Open a terminal.
 
 * Go the the directory where you would like this package to live.
 
-* Run `leanproject get tutorials`.
+* Run `git clone https://github.com/leanprover-community/mathematics_in_lean.git`.
+
+* Run `cd mathematics_in_lean`
+
+* Run `lake exe cache get`
 
 * Launch VS Code, either through your application menu or by typing
-  `code tutorials`. (MacOS users need to take a one-off
+  `code .`. (MacOS users need to take a one-off
   [extra step](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line)
    to be able to launch VS Code from the command line.)
 
 * If you launched VS Code from a menu, on the main screen, or in the File menu,
   click "Open folder" (just "Open" on a Mac), and choose the folder
-  `tutorials` (*not* one of its subfolders).
+  `mathematics_in_lean` (*not* one of its subfolders).
 
 * Using the file explorer on the left-hand side, explore everything you
-  want in `tutorials/src`.
-  See the [tutorials instructions](https://github.com/leanprover-community/tutorials/blob/master/README.md)
-  for advice about how to do the exercises in this project.
+  want in `MIL`.
+  See the [MIL instructions](https://github.com/leanprover-community/mathematics_in_lean/blob/master/README.md)
+  for advice about how to do the exercises in this book.
 
 ## Creating a Lean project
 
 We will now create a new project depending on mathlib. The following
 commands should be typed in a terminal.
 
 * Go to a folder where you want to create a project in a subfolder
-  `my_project`, and type `leanproject new my_project`. If you get an
-  error message saying `leanproject` is an unknown command and
-  you have not logged in since you installed Lean and mathlib, then
+  `my_project`, and type `lake +leanprover/lean4:nightly-2023-02-04 new my_project math`. If you get an
+  error message saying `lake` is an unknown command and
+  you have not logged in since you installed Lean, then
   you may need to first type `source ~/.profile` or `source ~/.bash_profile`.
 
```

Comment:
```suggestion
The keyword `math` at the end of this command adds `mathlib4` to the dependencies of your project, so that you can use `import Mathlib` in your project files.

```